DRsim/include/DRsimSteppingAction.hh

```diff
@@ -18,22 +19,15 @@ public:
   virtual void UserSteppingAction(const G4Step*);
 
 private:
+  const DRsimDetectorConstruction* DetectorConstruction;
```

Comment:
Minor comment - let's try to have a different annotation for the name of a class and a variable. The name `DetectorConstruction` more looks like a class name rather than a variable. Could you try other names e.g. `pDetectorConstruction` or `detectorConstruction`?

Reply:
Yes, I will check and change the name